### PR TITLE
#33584 filtering shebang line from --queries-file

### PR DESCRIPTION
**Pull request by egor-sivokhin2001**
Changelog category (leave one):
- New Feature

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Filtering `shebang` line from `--queries-file` to support shebang.

> Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by qoega** [comment]
@egor-sivokhin2001 please sign CLA.

Can you add a test case with this scenario?
What is the original idea? It is probably an SQL query passed in --query-file and what sort of shebang you encounter there?

**Comment by alexey-milovidov** [comment]
@qoega This is for #33584.

**Comment by alexey-milovidov** [comment]
@egor-sivokhin2001 Also please fix the style check:
https://s3.amazonaws.com/clickhouse-test-reports/34095/612f97f8e9240c0962fb314d4ae08f5ef70231c0/style_check__actions_.html

**Comment by alexey-milovidov** [comment]
Alternative implementation is also considered: #34138

File: src/Client/ClientBase.cpp [hunk 1]
```diff
@@ -1641,6 +1641,9 @@ void ClientBase::runNonInteractive()
             ReadBufferFromFile in(file);
             readStringUntilEOF(queries_from_file, in);
 
+            if (queries_from_file.starts_with("#!"))
```
Comment (alexey-milovidov) [inline src/Client/ClientBase.cpp:1644]: Need to add a comment.

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]
Also need to add a test, see tests/queries/0_stateless/ *.sh, *.reference

**Comment by alexey-milovidov** [comment]
The task has been automatically solved by #34230.
I will add a test.

**Pull request closed without merging**